tests/test_assets.py

```diff
@@ -1585,3 +1587,18 @@ def test_fuzzy(self):
             results,
             [af.lookup_symbol(sym, dt, fuzzy=True) for sym in syms],
         )
+
+
+class TestAssetFinderPreprocessors(WithTmpDir, ZiplineTestCase):
+
+    def test_asset_finder_doesnt_silently_create_useless_empty_files(self):
+        nonexistent_path = self.tmpdir.getpath('nothing_here')
```

Comment:
What do you think about using a unique name instead of 'nothing_here' to prevent any conflict between tests in this class?

Reply:
updated to prepend `self.id()`

Reply:
👍